addons/portal/views/portal_templates.xml

```diff
@@ -143,7 +143,11 @@
     <template id="portal_breadcrumbs" name="Portal Breadcrumbs">
         <ol t-if="page_name != 'home'" class="o_portal_submenu breadcrumb mb-0 flex-grow-1 px-0">
             <li class="breadcrumb-item ms-1"><a href="/my/home" aria-label="Home" title="Home"><i class="fa fa-home"/></a></li>
-            <li t-if="page_name == 'my_details'" class="breadcrumb-item">Details</li>
+            <li t-if="page_name == 'my_addresses'" class="breadcrumb-item">Addresses</li>
+            <li t-if="page_name == 'address_form'" class="breadcrumb-item">
+                <a href="/my/addresses" aria-label="Home" title="Home">Addresses</a>
+            </li>
+            <li t-if="page_name in ['my_details', 'address_form']" class="breadcrumb-item">Details</li>
```

Comment:
```suggestion
            <li t-elif="page_name in ['my_details', 'address_form']" class="breadcrumb-item">Details</li>
```

Reply:
I want to add `Details` breadcrum for address for also `Addresses/Details `so I have to keep it t-if so Addresses and Details both breadcrumb added for address_form